dev/readthedocs.md

```diff
@@ -4,30 +4,30 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
 answer the following questions.
 
 1. What is the command I run to get the courses from Fall 2021?
-
+`python manage.py ingest jhu --years 2021 --terms Fall`.
 
 2. How do I then load those courses into my database?
-
+`python manage.py digest jhu`.
 
 3. How do I get a terminal running in my docker container?
-
+`docker exec -it <container-id> /bin/bash` or use Docker Desktop.
 
```

Comment:
you can also download the docker vscode plugin and find your running container there and click on `attach shell`